include/code_08124BE0.h

```diff
@@ -0,0 +1,28 @@
+#ifndef GUARD_CODE_08124BE0_H
+#define GUARD_CODE_08124BE0_H
+
+#include "data.h"
+#include "global.h"
+#include "pause_world_map.h"
+#include "task.h"
+
+// TODO: Into pause_help.h
+struct Unk_08D6113C {
+    /* 0x0 */ void* unkSrc;
+    /* 0x4 */ void* unkDest;
+}; /* size = 0x8 */
```

Comment:
```suggestion
struct Unk_08D6113C {
    /* 0x0 */ const u32* unkSrc;
    /* 0x4 */ u32* unkDest;
}; /* size = 0x8 */
```
The addresses must be aligned for CpuSet